platform/viewer/pluginConfig.json

```diff
@@ -31,6 +31,10 @@
     {
       "packageName": "@ohif/extension-cornerstone-dicom-seg",
       "version": "3.0.0"
+    },
+    {
+      "packageName": "dicom-rtss",
```

Comment:
can we name this extension dicom-rt?

Reply:
@ohif/extension-dicom-rt

Reply:
Will do, I will make this modification along with the colors in the next commit.